src/pages/generate/index.tsx

```diff
@@ -57,6 +58,8 @@ export default function GeneratePage() {
   // desktop view
   return (
     <GenerationProvider>
+      <Meta title="Generate" description="Create and manage your generated images" deIndex />
```

Comment:
do we still need a description even if is a `deIndex`?

Reply:
good point. removing!